content/sphinx.md

```diff
@@ -21,35 +17,48 @@ RST syntax along the way.
 - Please write your questions in the collaborative HackMD document so that we can answer
   them and discuss them together after the group sessions.
 
+```{discussion} This lesson is built with Sphinx
+- [Source code](https://raw.githubusercontent.com/coderefinery/documentation/main/content/sphinx.md)
+- Try to compare the source code and the result side by side
+- However, note that in this example we have used Markdown whereas in the following we will
+  demonstrate reStructuredText which is more typical in combination with Sphinx
```

Comment:
Just wonder why we don't stick to markdown.

Reply:
I wouldn't mind moving to Markdown but it would mean also explaining MyST.

Reply:
I think it would be quite nice but may not fit in the schedule.  so it is probably OK as is.

Reply:
It's a good idea. Following up in #202. Might not happen this time but at least it's recorded for the next person preparing.